llvm/lib/Target/AIE/Utils/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@
 # (c) Copyright 2023-2024 Advanced Micro Devices, Inc. or its affiliates
 add_llvm_component_library(LLVMAIEUtils
   AIEBaseInfo.cpp
+  AIE2LoopUtils.cpp
```

Comment:
Very much appreciated ;-) Although I think it should just be AIELoopUtils. There's nothing AIE2 specific in it.